keep-ui/components/navbar/IncidentLinks.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { Subtitle } from "@tremor/react";
+import { LinkWithIcon } from "components/LinkWithIcon";
+import { DoorbellNotification} from "components/icons";
+import { Session } from "next-auth";
+import { Disclosure } from "@headlessui/react";
+import { IoChevronUp } from "react-icons/io5";
+import classNames from "classnames";
+
+type IncidentsLinksProps = { session: Session | null };
+
+export const IncidentsLinks = ({ session }: IncidentsLinksProps) => {
+  const isNOCRole = session?.userRole === "noc";
+
+  if (isNOCRole) {
+    return null;
+  }
+
+  return (
+    <Disclosure as="div" className="space-y-1" defaultOpen>
+      <Disclosure.Button className="w-full flex justify-between items-center p-2">
+        {({ open }) => (
+          <>
+            <Subtitle className="text-xs ml-2 text-gray-900 font-medium uppercase">
+              INCIDENTS
+            </Subtitle>
+            <IoChevronUp
+              className={classNames(
+                { "rotate-180": open },
+                "mr-2 text-slate-400"
+              )}
+            />
+          </>
+        )}
+      </Disclosure.Button>
+
+      <Disclosure.Panel as="ul" className="space-y-2 p-2 pr-4">
+          <li>
+            <LinkWithIcon
+              href="/incidents"
+              icon={DoorbellNotification}
+              isBeta={true}
+            >
+              <Subtitle>Incidents</Subtitle>
+            </LinkWithIcon>
+          </li>
+      </Disclosure.Panel>
```

Comment:
@VladimirFilonov wdyt if we show N first incidents on the side panel? I think it will:
1. be cool
2. be consistent with the demo we had until now (+we gonna "continue" from last week)